_data/accessibility-tests/tooltip.yml

```diff
@@ -0,0 +1,100 @@
+title: Tooltip
+component_status: pass
```

Comment:
Our tests currently show two `AA` failures. Is there a reason we aren't marking this as a failure for now?

Reply:
I left this as "passing" because the fixes are in the works and should be published in 3.8.1. For consistency, I just updated the test status for these failing items to `pass` since it looks like the fixes are in a good state to be merged in 3.8.1. I also added a note at the top of the PR description saying that this PR should not be merged until these test items are indeed passing.